server/admin/lookup.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
+//
```

Comment:
same functionality as before. just moved out of `server/index.js` and into its own file.